tests/integration/targets/dns_forward_nsg_info/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+- module_defaults:
+    group/infoblox.bloxone.all:
+      csp_url: "{{ csp_url }}"
+      api_key: "{{ api_key }}"
+  block:
+    # Create a random Auth Zone name to avoid conflicts
+    - ansible.builtin.set_fact:
+        forward_nsg_name: "test-forward-nsg-{{ 999999 | random | string }}.com."
+        tag_value: "site-{{ 999999 | random | string }}"
+
+    - name: Create a Forward NSG
+      infoblox.bloxone.dns_forward_nsg:
+        name: "{{ forward_nsg_name }}"
+        state: present
+      register: forward_nsg
+    - name: Get Information about the Forward NSG
```

Comment:
Add a new line here please !

Reply:
done

Reply:
This comment is yet to be addressed . 
We need a line between lines 16 and 17 to differentiate between creation and the several retrieval methods.